### PR TITLE
irrlamb: added recipe

### PR DESCRIPTION
**Pull request by AlwaysLivid**
This recipe is a WIP, due to the fact that I'm not really sure about how I should deal with this error:

image

### CODE REVIEW

File: games-simulation/irrlamb/irrlamb-1.0.1r613.recipe [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	irrlamb$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libfreetype$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	lib:libglu$secondaryArchSuffix
+	lib:libopenal$secondaryArchSuffix
+	lib:libvorbis$secondaryArchSuffix
+	lib:libx11$secondaryArchSuffix
```
Comment (extrowerk) [inline games-simulation/irrlamb/irrlamb-1.0.1r613.recipe]: The X11 port cannot draw. You can either port the game to use Haiku gui or abandon this.

**Comment by extrowerk** [comment]
Please do not use X11 libs in your ports, they are not useful as is, they are usable only if one want to build programs which draws on a remote X11 server running on other OS.

**Comment by alwayslivid** [comment]
> Please do not use X11 libs in your ports, they are not useful as is, they are usable only if one want to build programs which draws on a remote X11 server running on other OS.

Will keep that in mind, thanks for the remark. I'm not planning to port the game at this time.

**Comment by extrowerk** [comment]
Ok, i made a mistake: the project needs X11 just because it uses an in-tree irrlicht, which uses X11 on unixes, but we have an irrlicht port which uses SDL for output.

I just eliminated every references for irrlicht and added support to use the system one. The project builds fine but crashes at start.
So technically it is possible to build irrlamb on Haiku, but it is also not trivial.

**Comment by extrowerk** [comment]
Debug log: http://0x0.st/zhIi.report

**Comment by extrowerk** [comment]
AFAIK @threedeyes made the Minetest port, where i had similar problem as i tried to port it. Maybe he can give some hint why it crashing.

**Comment by alwayslivid** [comment]
> AFAIK @threedeyes made the Minetest port, where i had similar problem as i tried to port it. Maybe he can give some hint why it crashing.

I'm reopening the pull request.

**Pull request closed without merging**